entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Check if $DOMAIN is set
+if [ -z $DOMAIN ]; then
+	echo -e "You need to set \$DOMAIN variable at run time\n"
+	echo -e "For example: docker run -d -p 80:80 -p 443:443 -e DOMAIN=example.com\n"
+	exit 1
+else
+	# Install acme.sh
+	apt-get -qq update
+	apt-get install -qq \
+		cron \
+		openssl \
+		curl \
+		coreutils \
+		socat \
+		git
+	git clone https://github.com/Neilpang/acme.sh.git /root/acme.sh && \
+	cd /root/acme.sh && \
+	git checkout 2.7.8 && \
+	/root/acme.sh/acme.sh --install
+
+	# Generate SSL cert
+	/root/.acme.sh/acme.sh --issue --standalone -d ${DOMAIN} -d www.${DOMAIN}
```

Comment:
I don't see how adding `www` here is applicable for using `wss` with webminerpool.

Reply:
looks like this may have been handled in #62. linking here to close the loop